frontend/public/locales/zh/cluster-operator.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Name": "名称",
+  "Status": "状态",
+  "Version": "Version",
```

Comment:
I'm doubt why "Version" is still "Version" in Chinese language. I see it's translated to Japanese when language is Japanese.

Reply:
Thanks @yanpzhan - I'll ask the translation folks about it.

Reply:
"Version" is now updated in this PR: https://github.com/openshift/console/pull/7274/files#diff-54b5a8d337c5adfcfe4a8d54df3321e3521f79adbf6c76ce2d3c65b382d31fdcR4 for the batch of translations following this one.

Reply:
The translation folks sent over an updated file; it's now checked in commit #2!